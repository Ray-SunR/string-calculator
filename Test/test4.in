n a abc
la
wa

t c a fg
lc
wc

i d c
ld
wd
e d

e a
e c
q
