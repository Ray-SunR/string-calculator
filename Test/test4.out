3
abc
5
abcfg
5
abcfg
